cirq-core/cirq/experiments/xeb_fitting.py

```diff
@@ -12,6 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """Estimation of fidelity associated with experimental circuit executions."""
+import dataclasses
 from abc import abstractmethod, ABC
 from dataclasses import dataclass
```

Comment:
nit: remove this import and use `@dataclasses.dataclass` throughout.

Reply:
done